test/testdata/README.md

```diff
@@ -9,3 +9,50 @@ $ openssl req -key test/testdata/test_blob_private_key -x509 -days 3650 -out cer
 
 and then you replace the old `test/testdata/test_blob_cert.pem` with the new certificate.
 
+# Generate Certificates and Certificate chain for Attach test
+If the test/testdata/test_attach_leafcert.pem or test_attach_rootcert.pem or test_attach_certchain.pem expires than you can generate a new certificate and certificate using the given steps
+
+1. Generate a private key for Root certificate
```

Comment:
Can you surround code snippets in backticks (`openssl ...`), and put them on newlines? Check out the rendered view (https://github.com/sigstore/cosign/blob/6eb9fcc581c457759fdc9181cc5a103b4def0911/test/testdata/README.md) to see that the code snippets aren't on their own lines